napari/_qt/_qapp_model/_tests/test_view_menu.py

```diff
@@ -64,7 +65,8 @@ def test_toggle_axes_scale_bar_attr(
         * `ticks`
     """
     app = get_app_model()
-    viewer = make_napari_viewer()
+    init_qactions()
```

Comment:
This use to me suggests a refactor is needed to remove certain actions from qactions... Is that your perspective also @Czaki? Doesn't need to be in this PR, but I just want to make sure my understanding/spidey-sense is on the right track here...

Reply:
changed